components/gitpod-protocol/package.json

```diff
@@ -32,7 +32,7 @@
         "mocha": "^10.2.0",
         "rimraf": "^2.6.2",
         "ts-node": "^10.4.0",
-        "typescript": "~4.4.2",
+        "typescript": "^5.5.4",
```

Comment:
Hey @filiptronicek , this seems like a big jump, which could have unintended consequences. May I ask, is this required to support priority? Or is it a drive by change to update typescript?

Reply:
Yes, typescript on this version does not have the DOM type for `priority` in `RequestInit`.